crates/oxi-luvit/src/async.rs

```diff
@@ -49,14 +49,14 @@ impl AsyncHandle {
     /// this handle. It is safe to call this function from any thread. The
     /// callback will be called on the main thread.
     ///
-    /// NOTE: [libuv] will coalesce calls to [`AsyncHandle::send`], that is,
+    /// NOTE: [luvit] will coalesce calls to [`AsyncHandle::send`], that is,
     /// not every call to it will yield an execution of the callback. For
     /// example: if [`AsyncHandle::send`] is called 5 times in a row before the
     /// callback is called, the callback will only be called once. If
     /// [`AsyncHandle::send`] is called again after the callback was called, it
     /// will be called again.
     ///
-    /// [libuv]: https://libuv.org/
+    /// [luvit]: https://luvit.org/
```

Comment:
I just checked the site (hadn't heard of luvit and was curious) and found the domain isn't owned by anyone--is that expected?

Reply:
Oh, perhaps it should be https://luvit.io?

Reply:
Yes, that's just the result of grepping for `libuv` and substituting the results w/ `luvit`. I'll do a docs pass before merging.